views/administration/teams.hbs

```diff
@@ -10,24 +10,36 @@
         <script src="/scripts/administration/teams.js" type="text/javascript" defer></script>
     {{/content}}
     {{#content "page" mode="prepend"}}
-        <form action="/administration/teams/disablestudents/{{school._id}}" method="post" class="edit-form-school">
+		{{#ifConfig "STUDENT_TEAM_CREATION" "disabled"}}
```

Comment:
please add a default sentence that the users know about the setting and that it is disabled.